src/core/stdc/string.d

```diff
@@ -35,23 +35,23 @@ nothrow:
 @nogc:
 
 ///
-inout(void)* memchr(return inout void* s, int c, size_t n) pure;
+inout(void)* memchr(return inout void* ptr, int value, size_t n) pure;
 ///
-int   memcmp(scope const void* s1, scope const void* s2, size_t n) pure;
+int   memcmp(scope const void* dest, scope const void* src, size_t n) pure;
```

Comment:
I do not consider this change to be an improvement. Naming the parameters "source" and "destination" implies something is moving from the source to the destination.